jobmatch/src/main/resources/data.sql

```diff
@@ -0,0 +1,15 @@
+--
+-- Seed the Role database with possible user roles
+--
+MERGE INTO ROLE AS t USING (VALUES(1,'user'), (2, 'company')) AS vals(a,b)
```

Comment:
I see, so it's more of a sanity check. I'm totally into automatic repair like this; awesome!